src/stores/AppStore.ts

```diff
@@ -29,7 +29,10 @@ import {
   DeleteFileAction,
   UpdateFileNameAndDescriptionAction,
   LoadProjectAction,
-  LogLnAction
+  AddFilePreview,
+  LogLnAction,
+  RemoveFilePreview,
+  removeFilePreview
```

Comment:
no need to export removeFilePreview -- you can call store private method directly.